feature/topics/src/main/kotlin/uk/govuk/app/topics/ui/widget/TopicsWidget.kt

```diff
@@ -187,39 +187,39 @@ private fun TopicsWidgetPreview() {
     GovUkTheme {
         TopicsWidgetContent(
             topics = listOf(
-                TopicUi(
+                TopicItemUi(
                     "",
                     uk.govuk.app.design.R.drawable.ic_topic_default,
                     "A really really really really really really long topic title",
```

Comment:
It's a great idea to test this! But, we might even longer text - see screenshot where the description has completely disappeared and the title is chopped. There may well be even longer ones waiting in the wings too as there's no char limit as far as I know on this field in the content API. Could/should we chop it and give it an ellipsis, e.g. "Being diagnosed with a ...", shrink the font (I thinking this is not easy or desirable for accessibility), something else?

I'm guessing this design issue needs to be addressed, so you're waiting for a steer ❓ 

![Screenshot_20241028_085540](https://github.com/user-attachments/assets/ce1a0c40-2e3d-4f71-a3fa-05e184af6185)

Reply:
I was going to raise this in post amigo. It's mostly an issue with the header being a fixed height and also the design not taking titles that wrap into account. It looks rubbish when it wraps onto two lines even if text isn't getting cut off as the gap between the title and the content is too small (issue with using a fixed height rather than padding).

Reply:
Yeah, I figured you might let Post Amigo deal with this design issue - I think that's a wise move. Definitely need some guidance on this. It'll effect visited items and potentially other areas of the app too 👍